web-extensions/chrome/engine/bookmycomics.js

```diff
@@ -42,7 +42,7 @@ function BmcEngine(hostOrigin, readerName, comicInfo) {
                     retErr = new Error(LOGS.getString('E0010', {'label': evData.label,
                                                                 'err': err.message}));
                 }
-                this.sendNotificationWithComicInfo('Register Comic', retErr);
+                this.sendNotificationWithComicInfo('Register Comic', retErr, evData.label);
```

Comment:
Isn´'t the function expecting an `id` rather than a `label` ?

Reply:
Indeed but in this case, I don't know how to provide this id since it needs to be created...

Reply:
As discussed offline, we should probably reset the memoization, and register a .once callback on the memoization completion, which would call this method.

Reply:
Should I do it in this PR?